cfme/tests/intelligence/test_chargeback_assignments.py

```diff
@@ -14,124 +12,122 @@
 
 
 @pytest.mark.meta(blockers=[1273654])
-def test_assign_compute_enterprise(appliance, virtualcenter_provider):
+def test_assign_compute_enterprise(appliance, assigns_collection, virtualcenter_provider):
```

Comment:
Where is `assigns_collection` coming from? Does not seem like something that should be a fixture, as its referenced in other tests how I would expect to see it here, as `appliance.collections.chargeback_assignments`